### ISSUE TITLE
skyculture greek_leidenAratea : When search "Pleiades", dark screen was shown.

### ISSUE THREAD

**Issue by 3domennotry** [Stellarium/stellarium#3931]
### Steps to reproduce

1. select skyculture greek_leidenAratea
2. open Search window
3. input "Pleiades" and enter
![20241003_1-1](https://github.com/user-attachments/assets/6c9c993c-e0a5-4a5f-ad64-69204cd20ba6)
![20241003_1-2](https://github.com/user-attachments/assets/98aea170-d6a0-4894-9bba-35dab4a37437)

I think that There is a problem with the file "constellationship.fab".
Line 51
M45 0 1165

by the way

(install_dir)/skycultures/greek_leidenAratea/

constellation_names.eng.fab
Line 2
Line 37

description.en.utf8
Line 6
Line 27
Line 220

Ursa Ma"i"or and Canis Ma"i"or 
Is this correct? Latin spelling?

----------

### System
* Stellarium version: stellarium-24.3-qt5-win32.exe

* Operating system: Win7 32bit

(Sorry, I can not speak English very well.)

thank you.

**Comment by gzotti** [comment]
> Ursa Ma"i"or and Canis Ma"i"or
> Is this correct? Latin spelling?

yes. There was no separate J in the Latin alphabet.

**Comment by alex-w** [comment]
> I think that There is a problem with the file "constellationship.fab".
> Line 51
> M45 0 1165

Yes. 1165 is designation of the star in HR catalog, but we use HIP designations for constellation's attachments. I've fixed this typo + definition of this constellation (the constellation should have minimum 1 segment). See revision https://github.com/Stellarium/stellarium/commit/a47099bc18fd7e220626cde1fbddbddbb1dae3dd

**Comment by 3domennotry** [comment]
I have confirmed that this bug has been fixed in stellarium-24.4-qt5-win32.exe.
Thank you very much.

**Comment by 3domennotry** [comment]
This bug has been fixed in stellarium-24.4-qt5-win32.exe.
I have confirmed that this bug has been fixed in stellarium-25.1-qt5-win64.exe too.
Thank you very much.
